protected/humhub/modules/activity/widgets/views/activityStream.php

```diff
@@ -16,11 +16,13 @@
 
 ?>
 
-
-<div class="panel panel-default panel-activities">
+<div class="panel panel-default panel-activities" id="panel-activities">
+   <?= \humhub\widgets\PanelMenu::widget(['id' => 'panel-activities']); ?>
     <div class="panel-heading"><?= Yii::t('ActivityModule.widgets_views_activityStream', '<strong>Latest</strong> activities'); ?></div>
+   <div class="panel-body">
     <?= Html::beginTag('div', $options) ?>
         <ul id="activityContents" class="media-list activities" data-stream-content>
         </ul>
+      </div>
```

Comment:
I think this is a bit confusing. We open the div around `<ul id="activityContents">` with `<?= Html::beginTag('div', $options) ?>` and close it with a regular `</div>` while we close the div for `<div class="panel-body">` with `<?= Html::endTag('div')?>`. I would prefer to close a tag opened by `<?= Html::beginTag() ?>` with `<?= Html::endTag() ?>` and vice versa.

Reply:
Sadly this is the only way that I've gotten it to actually work on collapsing and advise offered would be much appreciated, thanks. :smiley:

Reply:
Sounds strange. What happens when u switch line 26/27?